experiment/kind-e2e.sh

```diff
@@ -48,7 +48,9 @@ install_kind() {
     mkdir -p "${TMP_DIR}/bin"
     # if we have a kind checkout, install that to the tmpdir, otherwise go get it
     if [[ $(go list sigs.k8s.io/kind) = "sigs.k8s.io/kind" ]]; then
-        env "GOBIN=${TMP_DIR}/bin" go install sigs.k8s.io/kind
```

Comment:
reasonable to me.
for reference https://github.com/kubernetes-sigs/kind/blob/0a6ceaec7a648fd46cc75119633d8c03c59b882f/hack/ci/e2e.sh#L44